### PR TITLE
DAT-3934 builder with draft templates

### PR DESCRIPTION
**Pull request by idradm**

### CODE REVIEW

File: extensions/wikia/TemplateClassification/TemplateClassification.hooks.php [hunk 1]
```diff
@@ -105,17 +103,17 @@ public function onEditPageShowEditFormFields( \EditPage $editPage, \OutputPage $
...
+			$out->addHTML( \Html::hidden( 'templateClassificationTypeNew', $types[ 'new' ],
+				[ 'autocomplete' => 'off' ] ) );
 
 			// add additional class to body for new templates in order to hide editor while template classification
 			// modal is visible and builder is available
 			if (
 				$wgEnablePortableInfoboxBuilderExt
 				&& $title->getArticleID() === 0
-				&& empty( $types['current'] && $types['new'] )
+				&& empty( $types[ 'current' ] ) && empty( $types[ 'new' ] )
```
Comment (RafLeszczynski) [inline extensions/wikia/TemplateClassification/TemplateClassification.hooks.php]: is `empty()` the only change except text formatting?

**Comment by RafLeszczynski** [comment]
Please check why the ad related unit test failed

File: extensions/wikia/TemplateClassification/TemplateClassification.hooks.php (hunk 1 above)
Comment (idradm) [inline extensions/wikia/TemplateClassification/TemplateClassification.hooks.php]: yes

**Comment by idradm** [comment]
Failing test should be covered by https://wikia-inc.atlassian.net/browse/SEO-334

File: extensions/wikia/TemplateClassification/TemplateClassification.hooks.php [hunk 2]
```diff
@@ -50,8 +48,8 @@ public function onArticleSaveComplete( \WikiPage $article, \User $user, $text, $
 		 * so we exit early to prevent polluting of the results.
 		 */
 		if ( empty( $typeNew )
-			|| $typeNew === \TemplateClassificationService::NOT_AVAILABLE
-			|| $typeNew === $typeCurrent
+			 || $typeNew === \TemplateClassificationService::NOT_AVAILABLE
+			 || $typeNew === $typeCurrent
```
Comment (dianafa) [inline extensions/wikia/TemplateClassification/TemplateClassification.hooks.php:52]: why this spaces?

File: extensions/wikia/TemplateClassification/TemplateClassification.hooks.php (hunk 1 above)
Comment (dianafa) [inline extensions/wikia/TemplateClassification/TemplateClassification.hooks.php]: can we move `&& empty( $types[ 'new' ] )` to a separate line and move this logic to a separate function?

File: extensions/wikia/TemplateClassification/TemplateClassification.hooks.php (hunk 2 above)
Comment (idradm) [inline extensions/wikia/TemplateClassification/TemplateClassification.hooks.php:52]: this is how auto formatter is set, in editor it's on the same column as "empty"

**Comment by RafLeszczynski** [comment]
LGTM

**Pull request merged**